modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/SingleSelect.scss

```diff
@@ -0,0 +1,4 @@
+.lfr_object-single-select {
```

Comment:
Hey @igor-franca, according to the [BEM pattern](https://getbem.com/naming/), we should have two underscores here: 

`lfr__object-single-select`